langchain4j-oracle-database/src/main/java/dev/langchain4j/store/embedding/oracle/OracleEmbeddingStore.java

```diff
@@ -586,6 +615,54 @@ public Builder tableName(String tableName) {
             return this;
         }
 
+        /**
```

Comment:
Haven't we settled on using lombok for the builder?

Reply:
I did not include lombok when I re-wrote the implementation. It's easier to understand how our code works when you can see the code. With lombok, you need to read several pages of documentation to understand how the code works (https://projectlombok.org/features/Builder).